app/src/main/java/com/github/mik629/android/fundamentals/di/modules/AppModule.kt

```diff
@@ -12,4 +19,18 @@ internal object AppModule {
     @Singleton
     fun provideAppContext(application: Application): Context = application.applicationContext
 
-}
+    @Provides
+    @Singleton
+    fun provideUpdateCacheWorkRequest(): WorkRequest =
+        PeriodicWorkRequest.Builder(UpdateCacheWorker::class.java, 2, TimeUnit.MINUTES)
```

Comment:
2, TimeUnit.MINUTES)?
min limit is 15

Reply:
I wanted once per day, 2 minutes is leftovers after testing